### ISSUE TITLE
crate deno = "v0.32.0" failed to publish to crates.io

### ISSUE THREAD

**Issue by ry** [denoland/deno#3873]
not sure what happened with the crates.io publish - the error is here https://github.com/denoland/deno/runs/423535827

```
error[E0308]: mismatched types
##[error]   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/dprint-plugin-typescript-0.3.0-alpha.5/src/configuration.rs:878:57
    |
878 |         line_width: get_value(&mut config, "lineWidth", global_config.line_width, &mut diagnostics),
    |                                                         ^^^^^^^^^^^^^^^^^^^^^^^^ expected `u32`, found enum `std::option::Option`
    |
    = note: expected type `u32`
               found enum `std::option::Option<u32>`
```

cc @dsherret

**Comment by dsherret** [comment]
@ry yeah, that's my fault. I didn't know the way cargo pulls in dependencies marked as alpha and have had breaking changes between my incrementing alpha builds. I'll switch back to properly versioning.

Will fix this soon.

**Comment by dsherret** [comment]
@ry upgrade to dprint-plugin-typescript `0.3.0-alpha.6` and this will be fixed (perhaps it will just pull in that version now?)

Again, my bad and sorry for the bother! I will read into how cargo resolves dependencies so it doesn't happen again.

**Comment by dsherret** [comment]
Oh actually, now all the configuration has moved to the `configuration` module so that import statement will need to be updated.

**Comment by ry** [comment]
fixed by #3874
